### PR TITLE
Fix copying shortcuts, small changes to ftp

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Fixes an issue where files are not copied if any is a shortcut
- Fixes an issue where folder shortcuts don't open in ColumnView

**Details of Changes**
Add details of changes here.
- Fixed an issue where files are not copied if any is a shortcut
- Fixed an issue where folder shortcuts did not open in ColumnView
- Process commands in parallel in the fulltrust process
- Delay loading the "new item" context menu after the rest of the stuff is loaded (and stop using "Task.Run().Result"). Rationale: after #6657 the "new item" menu is fetched from FTP but it does not make sense to load it before the sidebar etc.)

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: Files.Launcher/Program.cs [hunk 1]
```diff
@@ -170,7 +170,7 @@ private static void EndReadCallBack(IAsyncResult result)
             }
         }
 
-        private static async void Connection_RequestReceived(PipeStream conn, Dictionary<string, object> message)
+        private static async Task Connection_RequestReceived(PipeStream conn, Dictionary<string, object> message)
```
Comment (d2dyno1) [inline Files.Launcher/Program.cs:173]: `async void`?

Comment (gave92) [reply Files.Launcher/Program.cs:173]: This method I have to call with Task.Run() so needs to return a Task.

Comment (d2dyno1) [reply Files.Launcher/Program.cs:173]: Ah right, could you break down this function into `RequestReceivedFromConnection()` and call that async Task function in Connectio_RequestReceived()

File: Files.Launcher/Win32API.cs [hunk 2]
```diff
@@ -197,24 +199,31 @@ public static (string icon, string overlay) GetFileIconAndOverlay(string path, i
...
-                        overlayStr = Convert.ToBase64String(bitmapData, 0, bitmapData.Length);
+                        return (iconStr, null);
                     }
+
+                    var overlayIdx = shfi.iIcon >> 24;
+                    if (overlayIdx != 0)
+                    {
+                        var overlayImage = imageList.GetOverlayImage(overlayIdx);
+                        using var hOverlay = imageList.GetIcon(overlayImage, ComCtl32.IMAGELISTDRAWFLAGS.ILD_TRANSPARENT);
+                        if (!hOverlay.IsNull && !hOverlay.IsInvalid)
```
Comment (d2dyno1) [inline Files.Launcher/Win32API.cs:215]: Shouldn't there be `||` ?

Comment (gave92) [reply Files.Launcher/Win32API.cs:215]: Nope, hOverlay is only good if not null and not invalid

**Review by d2dyno1 — APPROVED** [review]
lgtm

**Pull request merged**